skimage/segmentation/tests/test_seeds.py

```diff
@@ -0,0 +1,32 @@
+import itertools as it
+import numpy as np
+from numpy.testing import assert_equal, assert_raises
+from skimage.segmentation import seeds
+from skimage._shared.testing import test_parallel
+
+
+@test_parallel()
+def test_color_2d():
```

Comment:
You take in so many different parameters, and check for all of the. You should really test many more keyword argument and "bad input" cases.


Especially since you take in and special case many different types of dtypes.